### PR TITLE
Fix: code quality issues

### PR DESCRIPTION
**Pull request by withshubh**
### Description
Hi :wave: I ran the [DeepSource](https://deepsource.io) static analyzer on the forked copy of this repo and found some [interesting code quality issues](https://deepsource.io/gh/withshubh/airtable.py/issues/). This PR fixes a few of them.

### Summary of fixes
- ~Removed unnecessary import alias~
- Fix dangerous default argument
- ~Remove implicit `object` from the base class~
- ~Refactor unnecessary `else` / `elif` when `if` block has a `return` statement~

### CODE REVIEW

**Comment by withshubh** [comment]
Hi @josephbestjames :wave: 

Please have a look at this PR! :sparkles:

**Comment by pcorpet** [comment]
This file (__init__.py) is going away soon.

**Comment by withshubh** [comment]
> This file (**init**.py) is going away soon.

Hey @pcorpet 

Fixed three more issues. :sparkles:

**Comment by withshubh** [comment]
Hey @josephbestjames :wave: 

Please have a look at this PR!

**Comment by withshubh** [comment]
@josephbestjames @pcorpet :wave: 
review this PR, please!

File: airtable/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 from __future__ import absolute_import
 from .airtable import (
-    Airtable as Airtable,
-    AirtableError as AirtableError,
-    IsNotInteger as IsNotInteger,
-    IsNotString as IsNotString,
-    Table as Table,
+    Airtable,
```
Comment (pcorpet) [inline airtable/__init__.py:3]: Please rebase

File: airtable/airtable.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def __str__(self):
         return self.message or self.__class__.__name__
 
 
-class Airtable(object):
+class Airtable():
```
Comment (pcorpet) [inline airtable/airtable.py:58]: We still support python2, so we're not ready to drop this

File: airtable/airtable.py [hunk 3]
```diff
@@ -82,15 +82,16 @@ def __request(self, method, url, params=None, payload=None):
                              headers=self.headers)
         if r.status_code == requests.codes.ok:
             return r.json(object_pairs_hook=self._dict_class)
-        else:
-            error_json = r.json().get('error', {})
-            raise AirtableError(
-                error_type=error_json.get('type', str(r.status_code)),
-                message=error_json.get('message', json.dumps(r.json())))
+        error_json = r.json().get('error', {})
+        raise AirtableError(
```
Comment (pcorpet) [inline airtable/airtable.py:86]: Please rebase, I think this change got in in another commit

**Review by pcorpet — CHANGES_REQUESTED** [review]

**Comment by withshubh** [comment]
Hi @pcorpet Done! :sparkles: 

Also, please consider having a look at these [issues](https://deepsource.io/gh/withshubh/airtable.py/issues/?category=recommended).

**Comment by pcorpet** [comment]
This PR now hasn't any interesting changes to add.

**Pull request closed without merging**